src/apps/fuzzers/fuzzerPolygonToCellsExperimentalGeodesic.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2023-2024 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** @file fuzzerPolygonToCellsExperimentalGeodesic.c
+ * @brief Fuzzes the experimental polygon-to-cells implementation in geodesic
+ *        mode.
+ */
+
+#include "aflHarness.h"
+#include "h3api.h"
+#include "polyfill.h"
+#include "polygon.h"
+#include "utility.h"
+
+typedef struct {
+    int res;
+    int numHoles;
+    // repeating: num verts, verts
+    // We add a large fixed buffer so our test case generator for AFL
+    // knows how large to make the file.
+    uint8_t buffer[1024];
+} inputArgs;
+
+const int MAX_GEODESIC_RES = 4;
+const int MAX_SZ = 100000;
+const int MAX_HOLES = 100;
+
+static int populateGeoLoop(GeoLoop *g, const uint8_t *data, size_t *offset,
+                           size_t size) {
+    if (size < *offset + sizeof(int)) {
+        return 1;
+    }
+    int numVerts = *(const int *)(data + *offset);
+    *offset = *offset + sizeof(int);
+    g->numVerts = numVerts;
+    if (size < *offset + sizeof(LatLng) * numVerts) {
+        return 1;
+    }
+    g->verts = (LatLng *)(data + *offset);
+    *offset = *offset + sizeof(LatLng) * numVerts;
+    return 0;
+}
```

Comment:
We could make a follow up issue to deduplicate the populateGeoLoop function between the three files its now in